corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -227,13 +228,14 @@ def entry_for_module(self, module):
                 from corehq.apps.app_manager.suite_xml.features.mobile_ucr import get_report_context_tile_datum
                 e.datums.append(get_report_context_tile_datum())
 
-            if form.requires_case() and using_inline_search:
+            if form.requires_case() and module_offers_search(module):
```

Comment:
I thought I'd made this comment before but I can't find it anywhere...

This still needs to check the `loads_registry_case` condition and not add the post if that is True. I think that would resolve https://dimagi-dev.atlassian.net/browse/QA-4923

You could refactor the usage of `using_inline_search` a bit as well but that's a nit.

Reply:
ah thanks change made here: [c3b936c](https://github.com/dimagi/commcare-hq/pull/32554/commits/c3b936c2a0d8af2dc618ccdacdd509e49d69f061) and refactor here: [f752075](https://github.com/dimagi/commcare-hq/pull/32554/commits/f7520752de39081c34160ac0b74f3986b91cf64f)